### PR TITLE
feat: adds AURA token

### PR DESCRIPTION
**Pull request by 0xRichter**
Adds Aura Finance AURA token

Ethereum: https://etherscan.io/token/0xc0c293ce456ff0ed870add98a0828dd4d2903dbf
Base: https://basescan.org/address/0x1509706a6c66ca549ff0cb464de88231ddbe213b
Optimism: https://optimistic.etherscan.io/address/0x1509706a6c66CA549ff0cB464de88231DDBe213B

### CODE REVIEW

**Comment by wbnns** [comment]
@hamdiallam 

Heya, it looks like CI/CD is failing on this PR with the following message (possibly a bug in the validator):
```
pnpm validate --datadir ./data --tokens "AURA"

> @eth-optimism/tokenlist@10.0.87 validate /Users/wbnns/Git/base/temp/token-aura
> tsx ./bin/cli.ts validate "--datadir" "./data" "--tokens" "AURA"

(node:25749) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("file%3A///Users/wbnns/Git/base/temp/token-aura/node_modules/.pnpm/tsx%403.12.6/node_modules/tsx/dist/loader.js", pathToFileURL("./"));'
(Use `node --trace-warnings ...` to show where the warning was created)
[ 'AURA' ]
/Users/wbnns/Git/base/temp/token-aura/src/generate.ts:102
    const l2ChainsForL1 = L1_STANDARD_BRIDGE_INFORMATION[chain].map(
                                                                ^


TypeError: Cannot read properties of undefined (reading 'map')
    at getBridges (/Users/wbnns/Git/base/temp/token-aura/src/generate.ts:102:65)
    at <anonymous> (/Users/wbnns/Git/base/temp/token-aura/src/generate.ts:43:34)
    at Array.map (<anonymous>)
    at <anonymous> (/Users/wbnns/Git/base/temp/token-aura/src/generate.ts:41:42)
    at Array.map (<anonymous>)
    at generate (/Users/wbnns/Git/base/temp/token-aura/src/generate.ts:33:6)
    at validate (/Users/wbnns/Git/base/temp/token-aura/src/validate.ts:328:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Command.<anonymous> (/Users/wbnns/Git/base/temp/token-aura/bin/cli.ts:25:21)

Node.js v18.19.0
 ELIFECYCLE  Command failed with exit code 1.
 ```

**Review by hamdiallam — DISMISSED** [review]

File: data/AURA/data.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      "ethereum": {
+        "address": "0xC0c293ce456fF0ED870ADd98a0828Dd4d2903DBF"
+      },
+      "optimism": {
+        "address": "0x1509706a6c66CA549ff0cB464de88231DDBe213B"
+      },
+      "base": {
+        "address": "0x1509706a6c66CA549ff0cB464de88231DDBe213B"
+      },
+      "nobridge": true
```
[the remark this answers is not in the document]
Comment (hamdiallam) [reply data/AURA/data.json:18]: the validator script thinks `nobridge` is a chain name and looking it up.

We can improve on our errors here

[the remark this answers is not in the document]
Comment (0xRichter) [reply data/AURA/data.json:18]: Ah my bad, thanks for the hint!

**Review by wbnns — APPROVED** [review]
ACK

@0xRichter Thanks for the updates and for building on Base 🔵

**Pull request merged**